### DISCUSSION TITLE
Key metrics in separate widget

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#16787] [How to]
YetiForce CRM 6.4 Service Pack v6

Hello,

Some way to see Key metrics separately, for example in a separate widget, not together with all the other Key metrics.

Example Current Key metrics
![image](https://user-images.githubusercontent.com/96985476/210371412-68f44f24-c88f-44ca-89f9-aa16789f8e8f.png)

I would like to be able to see separate Key metrics for various filters that I have, for example, in calendar, and that only those of calendars are shown in this Key metric

**Comment by krseguros** [comment]
Hello,

Any comments on my question?

See another example of what I mean, the following image is a Key metrics separately widget that I have in crm vtiger, in which I can have these indicators separated, example:
     Calls{Today}
     C. Assigned{month}
     C.Calls{month}
     C. Proposal{month}
     Closed-Won{month}
     C. Discarded{month}
     Closed-Lost
![image](https://user-images.githubusercontent.com/96985476/210442994-bf78915d-11ea-471e-b1ef-7528a2ca7c1e.png)

**Comment by rskrzypczak** [comment]
Check out the tabular type widget.

![image](https://user-images.githubusercontent.com/10221999/213760243-1ed106a3-283a-4351-ab27-eee17e5d7b08.png)
